src/localization/imu_packager.py

```diff
@@ -11,25 +14,46 @@ class ImuPackager:
     published by gazebo & publish to /imu/data, to replicate the behavior of imu_driver.py
     """
 
+    imu_pub: rospy.Publisher
```

Comment:
Where is this ImuPackager even used? I don't see it in the launch file but maybe I missed it. Also, it seems like a more fitting name is SimImuDriver or ImuSITL since it just exists to maintain the same IMU interface in sim

Reply:
it's in simulator.launch. I can rename it to SimImuDriver since it's now doing more than just merging imu msgs